src/postgres/9-updates.sql

```diff
@@ -114,5 +114,36 @@ CREATE TABLE IF NOT EXISTS code (
 DELETE FROM edprole_privilege
 WHERE privilege = 'FORM_CREATE' AND edprole_id IN ('a5b4947a-67d2-434e-9889-59c2fad39676', '804b335c-f191-4d26-9b98-1ec1cb62b97d');
 
--- The FORM_CREATE privilege should only be assigned to DAAC Data Managers.
+DELETE FROM edprole_privilege
```

Comment:
This privilege is (incorrectly) being used in the dashboard [here](https://github.com/eosdis-nasa/earthdata-pub-dashboard/blob/develop/app/src/js/utils/table-config/requests.js#L42-L44) to identify whether or not to disable the form action button. For now, these privileges can't be removed until that is fixed